rest-api-spec/schema.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
Can we move this file to `src/main/resources` ? I consider this more as part of the rest-api-spec then meta-data of the project/build.

Reply:
++ Happy to move to where you think is best

Reply:
done